ToDoList/src/ToDoList.API.Services/DependencyRegister.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Net.Http;
+using System.Web;
+
+using ToDoList.Contracts;
+using ToDoList.Contracts.Services;
+using ToDoList.DependencyInjection;
+
+namespace ToDoList.API.Services
```

Comment:
It is kinda of a custom to create a folder based on `.Contracts`' folder system in Cloud - there will be `Services` folder in the `Services` assembly and namespace goes to ToDoList.API.Services.Services, true, however, it is only visible in the assembly itself (since the implementation is `internal`) and flat structure is not maintainable in large scale. It also encourages most developers to introduce multiple folders with less generic names in the contract (and therefore implementing assemblies) and that makes the code (structure) more navigable…

The root of the assembly is then reserved for the "entry" point of the assembly - implementation of the `IDependencyRegister` in most of the cases. You can see the similar pattern in the WebApi itself - there is `web.config` and `package.config` - both non-code configuration files for respective services the project is using (or is used by) and then there is the `global.asax(.cs)` that server as the entry point for IIS. That's it, everything else is organized in folders…